inst/plumber/v1/endpoints.R

```diff
@@ -131,41 +131,6 @@ function(req) {
   pipapi::get_pip_version(lkup = lkups$versions_paths[[req$argsQuery$version]])
 }
 
-#* Return valid parameters
-#* @get /api/v1/valid-params
-#* @param parameter:[chr] Query parameter
-#* @param version:[chr] Data version. Defaults to most recent version. See api/v1/versions
-#* @param format:[chr] Response format. Options are of "json", "csv", or "rds".
-#* @serializer switch
-function(req) {
-  out <- pipapi::get_param_values(
-    req$argsQuery$parameter, lkup = lkups$versions_paths[[req$argsQuery$version]])
-  attr(out, "serialize_format") <- req$argsQuery$format
-  out
-}
```

Comment:
Do we really want to remove this one? I may be nice to return it to the end-user, no?